### PR TITLE
Updated 104 lab content for watsonx assistant

### PR DESCRIPTION
**Pull request by spansare**
Fixes | Relates to **[issue link]**

Content changes:

- Any content updates here

Code changes:

- Any code changes here

### CODE REVIEW

File: content/Watsonx/Assistant/104.md [hunk 1]
```diff
@@ -1,6 +1,7 @@
 ---
+title: '104: Analytics'
 timeToComplete: 30
-updated: 2023-03-31
+updated: 2024-02-06
 ---
```
Comment (r2raman) [inline content/Watsonx/Assistant/104.md:6]: Include quiz alert. 

<QuizAlert text='Heads Up! Quiz material will be flagged like this!' />

File: content/Watsonx/Assistant/104.md [hunk 2]
```diff
@@ -13,8 +14,7 @@ improving your assistant’s performance.
 ## Prerequisites
 
 Complete this section with the assistant you used in the 103 section **or** your own assistant. If you
-want to use your own assistant, you must [upload](https://cloud.ibm.com/docs/watson-assistant?topic=watson-assistant-admin-backup-restore#backup-restore-import) the following [Action skill](https://raw.githubusercontent.com/CloudPak-Outcomes/Watson-Asst-Lab/main/action-skills/begin-104-action-skill.json) which reflects the
-skill after completing section 103.
+want to use your own assistant, you must [upload](https://cloud.ibm.com/docs/watson-assistant?topic=watson-assistant-admin-backup-restore#backup-restore-import) the following [Action skill](https://raw.githubusercontent.com/CloudPak-Outcomes/Watson-Asst-Lab/main/action-skills/begin-104-action-skill.json) which reflects the skill after completing section 103, but note that this will overwrite your existing actions.
 
 ## Introducing Analytics
```
Comment (r2raman) [inline content/Watsonx/Assistant/104.md:20]: Include event numbers 1 and 2 like.  Analyze(1) and Draft(2)

File: content/Watsonx/Assistant/104.md [hunk 3]
```diff
@@ -32,33 +32,211 @@ Unless you thoroughly tested or experimented with your assistant as you complete
...
 ### Adding data to the Analyze page via Preview
 
-Imagine you are a builder who has finished writing a set of actions that address a new use case.
+Imagine you are a builder who has finished writing a set of actions that address a new use case. Or imagine you are a product manager overseeing a virtual assistant project, and a subject-matter expert has just let you know that they fine-tuned a few conversational flows in the assistant.
 
-You need a way to share your assistant with a testing or quality assurance (QA) team, so they can go test the updated actions before you deploy them to a live or production environment. In order to do so:
+
+You need a way to share your assistant with a testing or quality assurance (QA) team, so they can go test the updated actions before you deploy them to a live or production environment. 
 
 1. Go the the **Preview** page for your assistant.
```
Comment (r2raman) [inline content/Watsonx/Assistant/104.md:75]: Include event numbers for Preview and copy link to share.

File: content/Watsonx/Assistant/104.md [hunk 4]
```diff
@@ -32,33 +32,211 @@ Unless you thoroughly tested or experimented with your assistant as you complete
...
+
 ### Adding data to the Analyze page via Lendyr
 
-Open your assistant on the Lendyr using the link you created in the previous section to embed your assistant. Try out a few different conversations. To engage in an additional separate conversation with your assistant there should be a **Restart** button in upper-left menu.
+Conversation data is added to the Analytics page when your assistant engages in conversation with end users.
+
+Open your assistant on the Lendyr website (consult the **Preview your assistant on Lendyr Bank’s website** portion of section 2. Actions to open your assistant on the Lendyr website if you do not know how to do this) and try out a few conversations. To engage in an additional separate conversation with your assistant, you can restart the assistant by opening your custom Lendyr URL in a new tab or by following these instructions:
+
+1. Open a **Guided Tour** tile from the splash screen on your custom Lendyr.com website.
+
```
Comment (r2raman) [inline content/Watsonx/Assistant/104.md:50]: Change guided tour to Accessible & Scalable AI. There is a quiz for this guided tour. Also, include an alert for the quiz in this section.

File: content/Watsonx/Assistant/104.md [hunk 5]
```diff
@@ -32,33 +32,211 @@ Unless you thoroughly tested or experimented with your assistant as you complete
...
+> **Note**: Pay attention that you are adding the synonyms to the Investment option and not one of the other options, as your order of options may differ from the image below.
+
+![](./images/104/image-021.jpg)
+
+Now, try using the new synonyms. You can do this without leaving the action build window:
+
+![](./images/104/image-022.jpg)
+
+Congratulations! You have successfully used watsonx Assistant’s analytics to improve an action in your assistant.
```
Comment (r2raman) [inline content/Watsonx/Assistant/104.md:230]: You can customize the completion by stating the following. 

"Congratulations! You've reached the end of lab 104: Analytics. Click, [lab 105] to start the next lab.

Add any further instruction for this lab before congratulating for completion. Like 

Next, create a brand-new action called How do I find my investor number? Add location of investor number as one of the additional phrases. Invent your own brief answer to the question, either text or image. (You can easily find stock image URLs using Google Images.)

**Review by r2raman — CHANGES_REQUESTED** [review]

File: content/Watsonx/Assistant/104.md (hunk 1 above)
Comment (spansare) [reply content/Watsonx/Assistant/104.md:6]: added

File: content/Watsonx/Assistant/104.md (hunk 2 above)
Comment (spansare) [reply content/Watsonx/Assistant/104.md:20]: added

File: content/Watsonx/Assistant/104.md (hunk 3 above)
Comment (spansare) [reply content/Watsonx/Assistant/104.md:75]: added

File: content/Watsonx/Assistant/104.md (hunk 5 above)
Comment (spansare) [reply content/Watsonx/Assistant/104.md:230]: updated

File: content/Watsonx/Assistant/104.md (hunk 1 above)
Comment (r2raman) [reply content/Watsonx/Assistant/104.md:6]: Heads Up! Quiz material will be flagged like this!

Please add above line on top of the page.

**Review by r2raman — APPROVED** [review]

**Pull request merged**